symfony/phpunit-bridge/5.3/phpunit.xml.dist

```diff
@@ -13,11 +11,8 @@
         <ini name="error_reporting" value="-1" />
         <server name="APP_ENV" value="test" force="true" />
         <server name="SHELL_VERBOSITY" value="-1" />
-        <!-- If you use phpunit-bridge directly, you can control extra behavior -->
-        <!--
         <server name="SYMFONY_PHPUNIT_REMOVE" value="" />
-        <server name="SYMFONY_PHPUNIT_VERSION" value="8.5" />
-        -->
+        <server name="SYMFONY_PHPUNIT_VERSION" value="9.5" />
```

Comment:
Symfony 5.3 requires php 7.2. But phpunit 9.5 requires php 7.3. So PHP 7.2 users won't be able to run phpunit by default or am I misunderstanding this?

Reply:
PHP 7.2 is EOL so it shouldn't be maintained anymore IMHO